index.html

```diff
@@ -164,5 +164,18 @@ <h3 class="languages-title">Languages</h3>
             </div>
         </div>
     </section>
+    <div class= "contact-me">
+        <h2 class="Contact-me-title">Contact me</h2>
+        <p class="Contact-me-body">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+        <form class="Contact-form" action="https://formspree.io/f/mgebklpe
+        " method="post">
+            <input type="text" id="name" name="name" placeholder="Write your name here!" required />
+            <input type="email" id="email" name="email" placeholder="Write your e-mail here!" required />
+            <textarea name="user-massage" id="textarea" cols="30" rows="10" placeholder="Write something here" required></textarea>
```

Comment:
- [ ] Kindly make sure the Message field accepts up to 500 chars.   _maxlength="500"_